README.md

```diff
@@ -52,6 +52,19 @@ nextflow run ${CODE_ROOT}/pipelines/annotation_pipeline.nf \
   --mappings ${LATEST_MAPPINGS}
 ```
 
+### Conda
+
+You can also install CMAT using Conda.
+For example the following installs CMAT in a new environment called `cmat`, activates the environment, and prints usage:
+```bash
+conda create -n cmat -c conda-forge -c bioconda cmat
+conda activate cmat
+cmat
+```
+
+Note that with conda installation you can't invoke the pipelines directly via Nextflow, so you will need to use the corresponding `cmat` commands - e.g. `cmat annotate` instead of `nextflow run annotation_pipeline.nf`.
+All the same command line options apply.
```

Comment:
Should we make this our main mode of executing CMAT ?
It sounds more elegant and easier to document.

Reply:
Assuming you mean just here in the readme, yes probably, I was just being lazy... I'll update.

I think it would also work for our OT submissions, I wanted to try getting the commands to work outside of the conda install but we could honestly even just use the conda version on our cluster...

Reply:
Is there somewhere the full list of _corresponding `cmat` commands_ to use with conda?

Reply:
I've created #409 for generalising the wrapper script (and hence the short `cmat` commands), for now as you've suggested I've just included the corresponding `cmat` commands in the readme.